example/README.md

```diff
@@ -0,0 +1,23 @@
+## Twirpy Examples
+
+## Requirements
+You can install requirements for running the examples using `pip` :
+`pip install -r requirements.txt`
+
+## Running server
+You can run the Twirpy ASGI application using servers such as uvicorn.
+Example uvicorn command : `uvicorn --port 3000 server:app`
+
+## Running client
+After server has started, you can make request using the example client with the following command :
+`python client.py`
+
+Example output :
+```
+size: 12
+color: "black"
+name: "bowler"
+```
+
+## Generated code
```

Comment:
Add command used to generate

Reply:
Added installation doc for plugin as well as example command for generate